### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by danutibe07**
-  Connect React and Redux.
-  Apply React best practices and language style guides in code.
-

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,32 +1,275 @@
...
+  line-height: normal;
+  letter-spacing: normal;
+  color: #979197;
+}
+
+.book-items .title {
+  width: 12.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-family: 'Roboto Slab', serif;
```
Comment (franclobo) [inline src/index.css]: - [ ] Please correct the name of the font family, it is `RobotoSlab` to be similar to the Zeplin design 👇🏽 

image

File: src/index.css [hunk 2]
```diff
@@ -1,32 +1,275 @@
...
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.book-items .author {
+  width: 6.688rem;
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-family: 'Roboto Slab', serif;
```
Comment (franclobo) [inline src/index.css]: - [ ] Please correct the name of the font family, it is `RobotoSlab` to be similar to the Zeplin design 👇🏽 

image

File: src/index.css [hunk 3]
```diff
@@ -1,32 +1,275 @@
...
+  gap: 5px;
+  margin: 16px;
+  margin-left: 20rem;
+}
+
+.track button {
+  padding: 15px;
+  background-color: #0290ff;
+  color: #fff;
+  font-family: 'Roboto Slab', serif;
```
Comment (franclobo) [inline src/index.css]: - [ ] Please correct the name of the font family, it is `RobotoSlab` to be similar to the Zeplin design 👇🏽 

image

File: src/index.css [hunk 4]
```diff
@@ -1,32 +1,275 @@
...
+
+.chapters {
+  padding: 20px;
+}
+
+.Current-Chapter {
+  width: 9rem;
+  height: 1.125rem;
+  opacity: 0.5;
+  font-family: 'Roboto Slab', serif;
```
Comment (franclobo) [inline src/index.css]: - [ ] Please correct the name of the font family, it is `RobotoSlab` to be similar to the Zeplin design 👇🏽 

image

File: src/index.css [hunk 5]
```diff
@@ -1,32 +1,275 @@
...
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.Current-Lesson {
+  width: 13.25rem;
+  height: 1.313rem;
+  font-size: 1rem;
+  font-family: 'Roboto Slab', serif;
```
Comment (franclobo) [inline src/index.css]: - [ ] Please correct the name of the font family, it is `RobotoSlab` to be similar to the Zeplin design 👇🏽 

image

File: src/index.css [hunk 6]
```diff
@@ -1,32 +1,289 @@
...
+  padding: 15px;
+  justify-content: space-between;
+  border: solid 1px #e8e8e8;
+  margin: 20px 35px 0 35px;
+}
+
+.book {
+  display: flex;
+  flex-direction: column;
+  width: 350px;
```
Comment (franclobo) [inline src/index.css:95]: - **[OPTIONAL]** We could use width: 230px approx. to be more responsive 👇🏽 

https://user-images.githubusercontent.com/58642949/232227945-02c0dafd-a249-430f-9cab-301b3f9790e0.mp4

File: src/index.css [hunk 7]
```diff
@@ -1,32 +1,292 @@
...
+
+.navcontent {
+  display: flex;
+}
+
+.book-details {
+  display: flex;
+  background-color: #fff;
+  padding: 15px;
+  justify-content: space-between;
```
Comment (franclobo) [inline src/index.css:87]: - **[OPTIONAL]** We could use `justify-content: center;` to be more responsive 👇🏽 

https://user-images.githubusercontent.com/58642949/232227945-02c0dafd-a249-430f-9cab-301b3f9790e0.mp4

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @danutibe07, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Git flow ✔️ 
- Good UI ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

## PLEASE AVOID DOING CHANGES AFTER SUBMITTING YOUR PR, WHILE I WAS REVIEWING YOUR PR I NOTICED THAT YOU CONTINUE WORKING.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi @danutibe07, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**